problem_builder/mcq.py

```diff
@@ -167,6 +167,42 @@ def choice_name(choice_value):
                 self._(u"A choice value listed as correct does not exist: {choice}").format(choice=choice_name(val))
             )
 
+    def student_view_data(self):
+        """
+        Returns a JSON representation of the student_view of this XBlock,
+        retrievable from the Course Block API.
+        """
+        return {
+            'id': self.url_name,
```

Comment:
This should just be `self.name`, which comes from `QuestionMixin`. Please do not use `url_name`, and don't change `url_name` to return `name` in the workbench.